TesseractOCR/G8Tesseract.mm

```diff
@@ -785,7 +785,12 @@ - (NSData *)recognizedPDFForImages:(NSArray*)images {
 
 - (UIImage *)imageWithBlocks:(NSArray *)blocks drawText:(BOOL)drawText thresholded:(BOOL)thresholded
 {
-    UIImage *image = thresholded ? self.thresholdedImage : self.image;
+    UIImage *image = self.thresholdedImage;
+    if (!thresholded) {
+        // Have to rotate the original image to the right or it shows upside down
+        CGImageRef imageRef = self.image.CGImage;
+        image = [[UIImage alloc] initWithCGImage:imageRef scale:self.image.scale orientation:UIImageOrientationRight];
```

Comment:
It is possible that this problem only occurs for me. I would appreciate if someone can tell me what did I do wrong. I passed Tesseract an image with horizontal text. Tesseract was able to recognize the text. Only problem is when I ask for imageWithBlocks, it gives back an image upside down, when the blocks are at the right location, as shown in the first image. And if I say use threshold, everything is fine. So my feeling is that the orientation of the self.threshold image is correct but self.image is wrong universally.
I will look into how to add test code.